api/helpers/utils.js

```diff
@@ -347,6 +347,19 @@ exports.filterData = function (collection, data, roles) {
       delete item.reviewExtensions;
     });
     return data;
+  } else if (collection === 'Organization') {
+    _.each(data, function (item) {
+      console.log(item);
+      if (item.searchResults) {
+        for (let organization in item.searchResults){
```

Comment:
Nit-picky:
"organization" is actually an index, so maybe rename it to specify that it's not the organization object iteself? or optionally:

```
item.searchResults.forEach(organization => { 
    delete organization.description;
    delete organization.postal;
    delete organization.address1;
    delete organization.address2;
});
```